### ISSUE TITLE
有个问题麻烦请教下

### ISSUE THREAD

**Issue by WuYHH** [NLPWM-WHU/LSTPM#7]
1. layer_2_sims =  F.softmax(sessions_represent.bmm(layer_2_current).squeeze(2) * 1.0/avg_distance, dim = 1). 这行代码对应的是哪个公式呢？特别是avg_distance.

**Comment by aaaliluya** [comment]
> 1. layer_2_sims =  F.softmax(sessions_represent.bmm(layer_2_current).squeeze(2) * 1.0/avg_distance, dim = 1). 这行代码对应的是哪个公式呢？特别是avg_distance.

请问您找到数据集了吗？如果找到了可以共享吗
